pkg/konfluxgen/kustomize/fbc-builder.yaml

```diff
@@ -324,11 +312,7 @@ spec:
       operator: in
       values:
       - "false"
-    workspaces:
-    - name: workspace
-      workspace: workspace
   workspaces:
-  - name: workspace
```

Comment:
Since this is not needed anymore, we can remove this section from the PipelineRun that references this:

https://github.com/openshift-knative/hack/blob/5d3194a5542e6106601535b543d4c625624504e1/pkg/konfluxgen/pipeline-run.template.yaml#L84-L96